lib/ssl.js

```diff
@@ -67,13 +67,15 @@ Command.help = function() {
 `);
 };
 
+
```

Comment:
is your local dev environment running eslint + prettier on these files?

the built in eslint rules should prevent these extra spaces.

Reply:
i actually don't have any experience running those tools on a javascript environment so i'll check it out and see

Reply:
what do you use for an IDE?

Reply:
umm, vim.  I ran `npm run lint` on the project prior to the latest commits.